packages/firestore/e2e/Query/where.e2e.js

```diff
@@ -294,6 +294,24 @@ describe('firestore().collection().where()', function () {
     });
   });
 
+  it('returns with when combining greather than and lesser than on the same nested field', async function () {
+    const colRef = firebase.firestore().collection(`${COLLECTION}/filter/greaterandless`);
+
+    await Promise.all([
+      colRef.add({ nested: { field: 0 } }),
+      colRef.add({ nested: { field: 1 } }),
+      colRef.add({ nested: { field: 2 } }),
+      colRef.add({ nested: { field: 3 } }),
+    ]);
+
+    const snapshot = await colRef
+      .where(new firestore.FieldPath('nested.field'), '>=', 1)
```

Comment:
these are erroring with firestore is not a known symbol type errors so something about referencing the FieldPath this way isn't working. I am 100% certain this is just because you were having emulator / local-test-run issues though and you'll see it + fix it as soon as you get that going :-)

Reply:
Ah yes thanks :) I was indeed pushing this to get me running. I've added another test that uses the dot notation :)